### PR TITLE
Added codare.fun presence

### PR DESCRIPTION
**Pull request by baybeku**
codare.fun is a code sharing website for discord bots

image

image

image

### CODE REVIEW

File: websites/C/CodAre/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "url": "codare.fun",
+  "description": {
+    "tr": "CodAre'ın kod paylaşım sitesi.",
+    "en": "CodAre's code sharing website."
+  },
+  "service": "CodAre",
+  "version": "1.0.0",
+  "logo": "https://barbecue.is-pretty.cool/88A9JqM.png",
+  "thumbnail": "https://barbecue.is-pretty.cool/5vWBXSv.png",
```
Comment (Bas950) [inline websites/C/CodAre/dist/metadata.json:15]: upload to imgur

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/C/CodAre/dist/metadata.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
   },
   "service": "CodAre",
   "version": "1.0.0",
-  "logo": "https://barbecue.is-pretty.cool/88A9JqM.png",
-  "thumbnail": "https://barbecue.is-pretty.cool/5vWBXSv.png",
+  "logo": "https://imgur.com/f2U2Zd9",
+  "thumbnail": "https://imgur.com/9VV6XNf",
```
Comment (Bas950) [inline websites/C/CodAre/dist/metadata.json:15]: use the i.imgur links

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/C/CodAre/dist/metadata.json (hunk 2 above)
Comment (baybeku) [reply websites/C/CodAre/dist/metadata.json]: > use the i.imgur links

im using hizliresim instead of imgur because imgur is so slow in turkey

Comment (ririxichan) [reply websites/C/CodAre/dist/metadata.json]: I want mention CONTRIBUTING.md has this line and you need to agree with it:
```All images in the metadata file must be hosted on i.imgur.com. Using content hosted on the website is not permitted as they can change the paths and files unwillingly.```

File: websites/C/CodAre/dist/metadata.json [hunk 3]
```diff
@@ -11,8 +11,8 @@
   },
   "service": "CodAre",
   "version": "1.0.0",
-  "logo": "https://i.hizliresim.com/s4Dy17.png",
-  "thumbnail": "https://i.hizliresim.com/n85M42.png",
+  "logo": "https://i.imgur.com/f2U2Zd9",
```
Comment (Bas950) [inline websites/C/CodAre/dist/metadata.json]: https://i.imgur.com/f2U2Zd9.png

File: websites/C/CodAre/dist/metadata.json [hunk 4]
```diff
@@ -11,8 +11,8 @@
...
+  "thumbnail": "https://i.imgur.com/9VV6XNf",
   "color": "#3b1b8a",
```
Comment (Bas950) [inline websites/C/CodAre/dist/metadata.json]: https://i.imgur.com/9VV6XNf.png

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**